fabricexample/src/Examples/SpeedTest/SpeedTest.tsx

```diff
@@ -0,0 +1,352 @@
+        sweepAngle.value
+      );
+
+      return path;
+    });
+
+    return (
+      <Group>
+        <Path path={active} style="stroke" strokeWidth={strokeWidth} />
+        <Path
+          path={shadowPath}
+          style="stroke"
+          strokeWidth={strokeWidth * 1.5}
+          opacity={0.3}
+        >
+          <BlurMask blur={50} respectCTM={false} />
+        </Path>
+        <ArcGradient />
+      </Group>
+    );
+  };
+
+  const NegativePath = () => {
+    const color = backgroundColor;
+
+    const negativeArc = useDerivedValue(() => {
+      const path = Skia.Path.Make();
+      const r = _r - strokeWidth;
+      path.addArc(
+        {
+          x: cx - r,
+          y: cy - r,
+          width: r * 2,
+          height: r * 2,
+        },
+        startAngle,
+        -(360 - sweepAngle.value)
+      );
+
+      return path;
+    });
+
+    const circleR = _r + strokeWidth / 2;
+
+    return (
+      <Group>
+        <Path
+          path={negativeArc}
+          style="stroke"
+          strokeWidth={strokeWidth * 5}
+          color={color}
+        />
+        <Circle c={vec(cx, cy)} r={circleR * 1.5} color="transparent">
+          <Paint style="stroke" strokeWidth={circleR} color={backgroundColor} />
+        </Circle>
+      </Group>
+    );
+  };
+
+  const Needle = () => {
+    const topWidth = 10;
+    const bottomWidth = topWidth / 2;
+    const needleHeight = 100;
+
+    const path = useDerivedValue(() => {
+      const needlePath = Skia.Path.Make();
+
+      needlePath.moveTo(cx - topWidth, cy);
+      needlePath.lineTo(cx + topWidth, cy);
+      needlePath.lineTo(cx + bottomWidth, cy - needleHeight);
+      needlePath.lineTo(cx - bottomWidth, cy - needleHeight);
+
+      return needlePath;
+    });
+
+    const transform = useDerivedValue(() => {
+      const startAngleRadians = startAngle * (Math.PI / 180);
+      const currentAngleRadians =
+        (sweepAngle.value + startAngle) * (Math.PI / 180);
+      const rotate = currentAngleRadians - startAngleRadians * 2;
+
+      return [
+        {
+          rotate,
+        },
+      ];
+    });
+
+    return (
+      <Group transform={transform} origin={{ x: cx, y: cy }}>
+        <Path path={path} origin={{ x: cx, y: cy }}>
+          <LinearGradient
+            start={vec(cx, cy)}
+            end={vec(cx, cy - 130)}
+            colors={["rgba(255,255,255,0)", "rgba(255,255,255,1)"]}
+          />
+        </Path>
+      </Group>
+    );
+  };
+
+  const SpeedLabel = () => {
+    const paragraph = useDerivedValue(() => {
+      if (!customFontMgr) {
+        return null;
+      }
+
+      const paragraphStyle = {
+        textAlign: TextAlign.Center,
+      };
+
+      const textStyle: SkTextStyle = {
+        color: Skia.Color("white"),
+        fontFamilies: ["SfPro"],
+        fontSize: 42,
+      };
+
+      const text = `${speed.value.toFixed(2).toString()}`;
+
+      return Skia.ParagraphBuilder.Make(paragraphStyle, customFontMgr)
```

Comment:
here it's the same story than for path. SkParagraphBuilder as a `reset()` method. this allows you to create the paragraph builder outside the useDerivedValue and then you can do `.reset().pushStyleStyle.addText(text)` etc.
But if this is too complicated/abstract, I can do it in a later step.

Reply:
```
    const para = Skia.ParagraphBuilder.Make(paragraphStyle, customFontMgr)

    const paragraph = useDerivedValue(() => {
      const text = `${speed.value.toFixed(2).toString()}`;

      return para
        .reset()
        .pushStyle(textStyle)
        .addText(text)
        .build();
    }, []);
```

this results in `Property 'pushStyle' does not exist on type 'void'`. Do you have any ideas how to proceed?